src/udisksdaemonutil.c

```diff
@@ -73,6 +73,70 @@ void udisks_string_wipe_and_free (GString *string)
 }
 
 /**
+ * udisks_variant_lookup_binary:
+ * @dict: A dictionary #GVariant.
+ * @name: The name of the item to lookup.
+ * @out_text: (out): Return location for the binary text as #GString.
+ *
+ * Looks up binary data in a dictionary #GVariant and returns it as #GString.
+ *
+ * If the value is a bytestring ("ay"), it can contain arbitrary binary data
+ * including NUL values. If the value is a string ("s"), @out_text does not
+ * include the terminating NUL character.
```

Comment:
Please replace ``NUL`` with ``'\0'`` above. Maybe it's just me not knowing some macro, but it definitely confuses me because it's so similar to ``NULL``.